app/views/admin/info.html.erb

```diff
@@ -42,54 +42,59 @@ See COPYRIGHT and LICENSE files for more details.
 <%= call_hook(:view_admin_info_top) %>
 
 <%= render(AttributeGroups::AttributeGroupComponent.new) do |component|
-      if Setting.show_product_version?
-        core_link = linked_sha_reference(OpenProject::VERSION.core_sha, OpenProject::VERSION.core_url)
-        component.with_attribute(key: t(:label_core_build), value: core_link) if core_link.present?
-
-        product_link = linked_sha_reference(OpenProject::VERSION.product_sha, OpenProject::VERSION.product_url)
-        component.with_attribute(key: t(:label_product_version), value: product_link) if product_link.present?
-      end
-
-      component.with_attribute(key: t(:label_core_version), value: OpenProject::Info.versioned_name)
-      component.with_attribute(key: t(:label_database_version), value: @db_version)
-
-      if display_security_badge_graphic?
-        content = content_tag :object, nil, data: security_badge_url, type: "image/svg+xml"
-        content += link_to "",
-                           ::OpenProject::Static::Links[:security_badge_documentation][:href],
-                           title: t(:label_what_is_this),
-                           class: "security-badge--help-icon icon-context icon-help1",
-                           target: "_blank"
-        component.with_attribute(key: "", value: content)
-      end
-
-      if OpenProject::Database.version_deprecated?
-        value = <<~STR.html_safe
-      <p>
-        Starting with OpenProject 12.0, PostgreSQL 13 is required to use OpenProject.
+  if Setting.show_product_version?
+    core_link = linked_sha_reference(OpenProject::VERSION.core_sha, OpenProject::VERSION.core_url)
+    component.with_attribute(key: t(:label_core_build), value: core_link) if core_link.present?
+
+    product_link = linked_sha_reference(OpenProject::VERSION.product_sha, OpenProject::VERSION.product_url)
+    component.with_attribute(key: t(:label_product_version), value: product_link) if product_link.present?
+  end
+
+  component.with_attribute(key: t(:label_core_version), value: OpenProject::Info.versioned_name)
+  component.with_attribute(key: t(:label_database_version), value: @db_version)
+
+  if OpenProject::Database.version_deprecated?
+    value = render(Primer::Alpha::Banner.new(
+      scheme: :warning,
+      icon: :alert
+    )) do
+      <<~STR.html_safe
+        Starting with OpenProject 16.0, PostgreSQL 16 is required to use OpenProject.
```

Comment:
Here we mention that PG 16 is requried, but the update guide is to version 17?

Reply:
Yes, the minimum requirement is PG16